src/data/resolvers/colony.ts

```diff
@@ -220,15 +248,29 @@ export const getProcessedDomain = async (
 
   try {
     if (ipfsMetadata) {
-      const {
-        domainName: metadataDomainName = null,
-        domainColor = null,
-        domainPurpose = null,
-      } = JSON.parse(ipfsMetadata);
-
-      name = metadataDomainName;
-      color = domainColor;
-      description = domainPurpose;
+      const metadataVersion = getEventMetadataVersion(ipfsMetadata);
+      if (metadataVersion === 1) {
+        /*
+         * original metadata format
+         */
+        const {
+          domainName: metadataDomainName = null,
+          domainColor = null,
+          domainPurpose = null,
+        } = JSON.parse(ipfsMetadata);
+
+        name = metadataDomainName;
+        color = domainColor;
+        description = domainPurpose;
```

Comment:
Aside from the `JSON.parse(ipfsMetadata)`, couldn't these lines could be written as the lines below? E.g. 

```
const domainMetadata = JSON.parse(ipfsMetadata);
name = domainMetadata?.domainName || null; 

etc...
```